src/frame/reason.rs

```diff
@@ -59,6 +59,8 @@ impl Reason {
     pub const INADEQUATE_SECURITY: Reason = Reason(12);
     /// The endpoint requires that HTTP/1.1 be used instead of HTTP/2.
     pub const HTTP_1_1_REQUIRED: Reason = Reason(13);
+    /// The connection was closed
+    pub const CONNECTION_CLOSED : Reason = Reason(14);
```

Comment:
These reasons are mapped to http 2.0 reasons from the spec. In this case, it would either be a protocol error or an I/o error (connection reset by peer).